.github/workflows/markdown-lint.yml

```
@@ -0,0 +1,43 @@
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Install Dependencies
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y ripgrep
+
+      - name: Run Markdown Lint Script
+        shell: bash
+        run: |
```

actionlint

shellcheck reported issue in this script: SC2035:info:3:32: Use ./*glob* or -- *glob* so names with dashes won't become options

(shellcheck)